test/test_integration_resultset.rb

```diff
@@ -118,7 +118,7 @@ def test_enumerable
   end
 
   def test_types
-    assert_equal [ "integer", "text" ], @result.types
+    assert_equal [ "integer", "text" ], @result.types.map { |t| t.downcase }
```

Comment:
Why do we need to downcase here if it's already handled in `pragmas.rb`?

Reply:
@casperisfine Because the `pragmas.rb` change only impacts the `table_info` call.  The `types` call here goes directly to the extension.

Reply:
My bad. Can we add a similar `downcase` in there to preserve backward compatiblity?

Reply:
@casperisfine I made that change.  Please take a look and let me know what you think.